src/EnsembleKalmanInversion.jl

```diff
@@ -80,10 +141,15 @@ function update_ensemble!(
     # G is deterministic
     y = deterministic_forward_map ? (ekp.obs_mean .+ noise) : (ekp.obs_mean .+ zero(noise))
 
-    # N_obs × N_obs \ [N_obs × N_ens]
-    # --> tmp is [N_obs × N_ens]
-    tmp = (cov_gg + scaled_obs_noise_cov) \ (y - g)
-    u += (cov_ug * tmp) # [N_par × N_ens]
+    if isnothing(failed_ens)
+        _, failed_ens = split_indices_by_success(g)
+    end
+    if !isempty(failed_ens)
+        @info "$(length(failed_ens)) particle failure(s) detected. Handler used: $failure_handler_method."
```

Comment:
```suggestion
        @info "$(length(failed_ens)) particle failure(s) detected. Handler used: $(nameof(FM))."
```
This may be more human-readable